src/HttpServer.cpp

```diff
@@ -335,10 +336,10 @@ void HttpServer::accept(const std::atomic_bool& terminate_flag) {
     arg->workersCounter = &_workersCounter;
 
     /*
-     * We need to increase workersCounter in the parent process to prevent race
-     * condition. Otherwise it's possible that parent process will terminate
-     * before child process increments a counter.
-     */
+   * We need to increase workersCounter in the parent process to prevent race
+   * condition. Otherwise it's possible that parent process will terminate
+   * before child process increments a counter.
+   */
```

Comment:
Same question as above. That's just not very pretty. Perhaps you could rewrite it somehow to make your version of clang-format happy?